exporters/otlp/BUILD

```diff
@@ -268,7 +271,6 @@ cc_library(
         ":otlp_grpc_client",
         "//ext:headers",
         "//sdk/src/logs",
-        "@com_github_opentelemetry_proto//:logs_service_proto_cc",
```

Comment:
Is service proto not needed for grpc exporter?

Reply:
It's already depended by logs_service_grpc_cc below.